blivet/devices/lvm.py

```diff
@@ -792,6 +792,10 @@ def _preDestroy(self):
         # set up the vg's pvs so lvm can remove the lv
         self.vg.setupParents(orig=True)
 
+        # setting up VG's PVs may have caused this LV was automatically
+        # activated, make sure it is deactivated before removal
+        self._teardown()
```

Comment:
Because the code tears the device down before calling ``destroy()`` and we really only want to call ``lvdeactivate()`` after the other invocations in ``_pre_destroy()``.